src/Console/Command/AttachmentMigrateCommand.php

```diff
@@ -169,9 +169,25 @@ private function moveFile($file): void
         $file_path = AttachmentManager::generatePath($iaf_id, $filename, $issue_id);
         $new_path = str_replace("{$this->sm->getDefaultAdapter()}://", "{$this->target_adapter}://", $file_path);
 
-        // throws League\Flysystem\Exception
-        // we let it abort whole process
-        $this->sm->moveFile($old_path, $new_path);
+        // Check if file exists. If it does
```

Comment:
```suggestion
        // Check if file exists. If it does,
```

missing comma in the sentence